packages/showcases/css/stories/components/navigation/tabs/examples/demo.html

```diff
@@ -1,9 +1,8 @@
 <div class="block">
   <nav id="demo-tabs-nav" aria-label="Tabs list" class="vtmn-tabs">
     <ul id="demo-tabs-tablist" role="tablist">
-      <span id="block-focus-trap-start" tabindex="-1"></span>
       <li id="tab-demo-1" role="tab" aria-selected="false">
-        <a href="" class="selected">Tab Name</a>
+        <a href="">Tab Name</a>
```

Comment:
Not a fan of empty href, shouldn't we use `button` tags or set the `tabpanel` id here ?